### PR TITLE
Remove squid pizzly

### PR DESCRIPTION
**Pull request by rannick**
Removing squid and pizzly as fusion detection tools

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/multiqc_config.yaml [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+  "nf-core-rnafusion-summary":
+    order: -1002
 
 export_plots: true
+
+run_modules:
+  - fastqc
+  - qualimap
+  - picard
+  - mosdepth
```
Comment (fevac) [inline assets/multiqc_config.yaml:23]: Remove `mosdepth` and `qualimap` if they are removed and merge before this PR

Comment (fevac) [reply assets/multiqc_config.yaml:23]: We also have data for STAR and FASTP (if used). IMO they should also be included

File: assets/multiqc_config.yaml [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
+
+module_order:
+  - fastqc:
+      name: "FastQC"
+      path_filters:
+        - "*.zip"
+  - picard:
+      name: "Picard"
+  - qualimap:
```
Comment (fevac) [inline assets/multiqc_config.yaml:32]: remove?

File: assets/multiqc_config.yaml [hunk 3]
```diff
@@ -1,11 +1,40 @@
...
+      name: "Qualimap"
+  - mosdepth:
+      name: "Mosdepth"
+
+extra_fn_clean_exts:
+  - "_sorted_md"
+  - type: regex
+    pattern: "_T[0-9]"
```
Comment (fevac) [inline assets/multiqc_config.yaml:40]: where is this pattern used?

File: CHANGELOG.md [hunk 4]
```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- Remove `squid` and `pizzly` fusion detection tools [#406](https://github.com/nf-core/rnafusion/pull/406)
```
Comment (fevac) [inline CHANGELOG.md:16]: This PR also:
- Changes multiqc template
- change picard module to gatk4

File: conf/modules.config [hunk 5]
```diff
@@ -162,14 +153,10 @@ process {
 
     }
 
-    withName: PICARD_MARKDUPLICATES {
+    withName: GATK4_MARKDUPLICATES {
```
Comment (fevac) [inline conf/modules.config:156]: Let's test this with the big dataset

File: containers/arriba/Dockerfile
Comment (fevac) [inline containers/arriba/Dockerfile:1]: why is the arriba dockerfile removed? was this ever used?

File: containers/fusioncatcher/Dockerfile
Comment (fevac) [inline containers/fusioncatcher/Dockerfile:1]: same as before

**Review by fevac — CHANGES_REQUESTED** [review]
Awesome! 🌟 We also need to update the metro map image

File: containers/arriba/Dockerfile
Comment (rannick) [reply containers/arriba/Dockerfile:1]: it was used in the old versions of nextflow/nf-core, not used anymore

File: assets/multiqc_config.yaml (hunk 2 above)
Comment (rannick) [reply assets/multiqc_config.yaml:32]: Added to https://github.com/nf-core/rnafusion/pull/407

File: assets/multiqc_config.yaml (hunk 1 above)
Comment (rannick) [reply assets/multiqc_config.yaml:23]: Added to https://github.com/nf-core/rnafusion/pull/407

File: assets/multiqc_config.yaml (hunk 3 above)
Comment (rannick) [reply assets/multiqc_config.yaml:40]: This is a blooper from trying different things, removing

File: CHANGELOG.md (hunk 4 above)
Comment (rannick) [reply CHANGELOG.md:27]: Now picard to gatk4 is merged and multiqc template is changed in https://github.com/nf-core/rnafusion/pull/407

File: containers/fusioncatcher/Dockerfile
Comment (rannick) [reply containers/fusioncatcher/Dockerfile:1]: Removed containers

File: conf/modules.config (hunk 5 above)
Comment (rannick) [reply conf/modules.config]: Tested and merged

File: tower.yml [hunk 6]
```diff
@@ -18,11 +18,11 @@ reports:
...
     display: "Picard: Metrics from CollectRnaMetrics"
   "**/picard/*_rna_metrics.txt":
-    display: "Picard: Metrics from MarkDuplicates"
-  "**/qualimap/qualimapReport.html":
-    display: "Qualimap HTML report from STAR_FOR_STARFUSION alignment"
-  "**/qualimap/rnaseq_qc_results.txt":
-    display: "Qualimap QC results from STAR_FOR_STARFUSION alignment in TXT format"
+    display: "GATK4: Metrics from MarkDuplicates"
+  "**/picard/*insert*size*metrics.txt":
+    display: "GATK4: Metrics from InsertSizeMetrics"
```
Comment (fevac) [inline tower.yml:23]: If I understood what you meant earlier, I believe this is Picard for all except markduplicates

**Review by fevac — APPROVED** [review]
🚀 !!

**Pull request merged**